### PR TITLE
Add image gallery page.

### PR DESCRIPTION
**Pull request by asam01**
Change contact page to include images.
Minor changes to wording in other pages.

https://screenshot.googleplex.com/8LsHXB2t8cS.png - initial view of gallery
https://screenshot.googleplex.com/q777h5eWqwX.png - after clicking an image

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -20,11 +20,21 @@ function getRandomFact() {
...
-
   // Pick a random greeting.
   const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+//dealing with image gallery
```
Comment (yukyhaha) [inline portfolio/src/main/webapp/script.js:31]: nit: Add a space between double slash and dealing.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -33,4 +33,63 @@ h1 {
...
+  opacity: 0.8;
+  cursor: pointer;
+}
+
+.column img:hover {
+  opacity: 1;
+}
+
+.row:after {
+  content: "";
```
Comment (yukyhaha) [inline portfolio/src/main/webapp/style.css:69]: nit: get the fields ordered alphabetically, that's usually the best practice we have in the codebase

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -33,4 +33,63 @@ h1 {
...
+
+/*image gallery styling */
+
+/* grid: Four equal columns that floats next to each other */
+.column {
+  float: left;
+  width: 25%;
+}
+
+/* Style the images inside the grid */
```
Comment (yukyhaha) [inline portfolio/src/main/webapp/style.css:58]: you can nest those two fields in to .column for a easier reading

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (asam01) [reply portfolio/src/main/webapp/script.js:31]: Okay, will fix those throughout.

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (asam01) [reply portfolio/src/main/webapp/style.css:58]: Sorry, which fields are you referring to?

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (asam01) [reply portfolio/src/main/webapp/style.css:69]: will reorder them.

**Comment by asam01** [comment]
Fixes on new pull request.

**Pull request closed without merging**